python/py-dnspython/Portfile

```diff
@@ -26,7 +26,10 @@ checksums           rmd160  666b591e3b2d1a1201ffe44deb101966b63c4cf0 \
                     size    281325
 
 if {${name} ne ${subport}} {
-  depends_build-append port:py${python.version}-setuptools
+  depends_build-append port:py${python.version}-setuptools \
+                       port:py${python.version}-pip \
+                       port:py${python.version}-wheel \
+                       port:py${python.version}-setuptools_scm
```

Comment:
this should probably just use "python.pep517 yes" instead of adding the dependencies by hand

Reply:
Looks like @Schamschula resolved this. Closing this PR.